item-spec/examples/sentinel2-sample.json

```diff
@@ -55,6 +55,15 @@
       ]
     ]
   },
+  "properties": {
+    "collection": "s2-l2a",
```

Comment:
This is a top-level field.

Reply:
Cool, will change it. This does make me think we probably need to review all the examples? I only did minor tweaks on this one, most of your catches were on things I did not change (I did move properties up for more visibility, but didn't really change anything with it).

Reply:
Yes, before 1.0 we should definitely run all examples through validation (#298  / #445)